src/gui/search/searchtab.h

```diff
@@ -31,21 +31,25 @@
 #ifndef SEARCHTAB_H
 #define SEARCHTAB_H
 
-#include <QWidget>
+#include "searchsortmodel.h"
+#include "ui_searchtab.h"
 
+#define ENGINE_URL_COLUMN 4
+#define URL_COLUMN 5
+
+QT_BEGIN_NAMESPACE
```

Comment:
Don't use this. I already wrote somewhere here that it is meaningless (I mean, this is still not enough, so there is no need to clutter up the code).